src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFrom-SddlString.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using System.Security.AccessControl;
+using System.Security.Principal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Class comment
+    /// </summary>
+    [Cmdlet(VerbsData.ConvertFrom, "SddlString", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=623636", RemotingCapability = RemotingCapability.None)]
+    public sealed class ConvertFromSddlStringCommand : PSCmdlet
+    {
+        /// <summary>
+        /// The string representing the security descriptor in SDDL syntax
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0, ValueFromPipeline = true)]
+        public string Sddl { get; set; }
+
+        /// <summary>
+        /// The type of rights that this SDDL string represents, if any.
+        /// </summary>
+        [Parameter()]
+        [ValidateSet(
+            "FileSystemRights", "RegistryRights", "ActiveDirectoryRights",
+            "MutexRights", "SemaphoreRights", "CryptoKeyRights",
+            "EventWaitHandleRights")]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Implements the BeginProcessing method for the ConvertFrom-SddlString command.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+#if CORECLR
+            if (Type == "CryptoKeyRights" || Type == "ActiveDirectoryRights")
+            {
+                string errorMessage = StringUtil.Format(UtilityResources.TypeNotSupported, Type);
+                ErrorRecord errorRecord = new ErrorRecord(
+                    new ArgumentException(errorMessage),
+                    "TypeNotSupported",
+                    ErrorCategory.InvalidArgument,
+                    null);
+
+                ThrowTerminatingError(errorRecord);
+            }
+#endif
+        }
+
+        /// <summary>
+        /// Implements the ProcessRecord method for the ConvertFrom-SddlString command.
+        /// </summary>
+        protected override void ProcessRecord()
+        {
+            CommonSecurityDescriptor rawSecurityDescriptor = new CommonSecurityDescriptor(false, false, Sddl);
+
+            string owner = ConvertToNtAccount(rawSecurityDescriptor.Owner); 
+            string group = ConvertToNtAccount(rawSecurityDescriptor.Group);
+            List<string> discretionaryAcl = ConvertToAceString(rawSecurityDescriptor.DiscretionaryAcl, Type);
+            List<string> systemAcl = ConvertToAceString(rawSecurityDescriptor.SystemAcl, Type);
+
+            PSObject result = new PSObject();
+
+            result.Properties.Add(new PSNoteProperty("Owner", owner));
```

Comment:
Following up on my own question and after playing around with this cmdlet a little, we definitely want strongly typed output.